src/Checkout/Checkout.js

```diff
@@ -1,3 +1,7 @@
+import Form from "./components/Form";
+
 export function Checkout() {
-  return <div>Checkout</div>;
+  return <div>
+          <Form />
```

Comment:
something wrong with indentation. And check it in all the files and code, because there are some places below when it's broken.